apache-pulsar.yaml

```diff
@@ -0,0 +1,123 @@
+package:
+  name: apache-pulsar
+  version: 4.0.1
+  epoch: 0
+  description: Pulsar is a distributed pub-sub messaging platform with a very flexible messaging model and an intuitive client API.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-17-default-jvm
+      - openjdk-17-jre
```

Comment:
`-default-jvm` will pull in `-jre`, we should probably use `-jre` and install `-default-jvm` at the image level

Reply:
So I took out the `-default-jvm` leaving just the `-jre` and I get the common `Error: JAVA_HOME not set, and no java executable found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin` - I assume that's expected now until we install the `-default-jvm` in the image or specify it as part of what we need for testing?

Reply:
Yep, you'd either export `JAVA_HOME` or install `-default-jvm`